src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/HttpContextCurrentAnalyzer.cs

```diff
@@ -142,9 +143,20 @@ private static bool TryMatchSymbol(SyntaxNodeAnalysisContext context, SyntaxNode
                     return false;
                 }
             }
+            else if (accessedSymbol is IPropertySymbol propSymbol)
```

Comment:
This looks like a bugfix to be called out in the changelog.

Reply:
Good call. I'll update the changelog.